apps/evans/components/chrome/Footer.tsx

```diff
@@ -4,97 +4,93 @@ import NextBgImage from 'next-bg-image';
 import Link from 'next/link';
 import { t } from '@friends-library/locale';
 import type { Lang } from '@friends-library/types';
-import FriendsLogo from '../core/LogoFriends';
-import AmigosLogo from '../core/LogoAmigos';
+import Logo from '../core/Logo';
```

Comment:
loading both language logos was bloating our bundle. so i switched to generating a git-ignored "current" and "alt" language logo file. these are generated by default on `pnpm i` so CI doesn't give us errors, and in the just file, depending on what language we're invoking, they are switched around.